src/perf_tool_backend.ml

```diff
@@ -181,16 +187,25 @@ module Perf_line = struct
         in
         let src_instruction_pointer = int64_of_hex_string src_instruction_pointer in
         let dst_instruction_pointer = int64_of_hex_string dst_instruction_pointer in
-        let parse_symbol_and_offset str =
+        let parse_symbol_and_offset str ~addr =
           match Re.Group.all (Re.exec symbol_and_offset_re str) with
-          | [| _; symbol; offset |] -> symbol, Int.Hex.of_string offset
-          | _ | (exception _) -> "[unknown]", 0
+          | [| _; symbol; offset |] -> Symbol.From_perf symbol, Int.Hex.of_string offset
```

Comment:
It's a little strange that `perf` isn't resolving these symbols :)

Let's leave a comment here to remind ourselves of this weirdness later.

Reply:
Done